src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap

```diff
@@ -193,7 +193,7 @@ exports[`StoryPromo Container should render correctly for amp 1`] = `
     >
       <amp-img
         alt="Image Alt text"
-        attribution="Image provider"
+        attribution=""
```

Comment:
What happened to this?

Reply:
Copyright was not passed to `FigureContainer`. Fixed it.